packages/server/utils/ensureJiraDimensionField.ts

```diff
@@ -64,6 +64,7 @@ const ensureJiraDimensionField = async (
         jiraDimensionFields: team('jiraDimensionFields')
           .default([])
           .union(fieldsToAdd)
+          .orderBy((jiraDimensionField) => jiraDimensionField.coerceTo('string'))
```

Comment:
can you verify if this stringifier is stable in the docs or github issue? if not, we might want to union `[...newJiraDimensionFields, ...fieldsToAdd]` in JS land to guarantee order

Reply:
I couldn't find anything quick so I did the union & ordering in JS land. This is also consistent with `updateJiraDimensionField.ts ` so I like it better.